### PR TITLE
Added RFC 5 - Signature Verfification for JWTs

### PR DESCRIPTION
**Pull request by dragonmantank**
Updates [RFC 1](https://github.com/Vonage/server-sdk-specification/blob/main/rfcs/accepted/RFC-1-server-sdk-jwt-packages.md) with a new "Signature Verification" section. Everything else stays the same.

Please comment below, or add a +1 to vote in favor.

### CODE REVIEW

File: rfcs/proposed/RFC-5-server-sdk-jwt-packages.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+JWTs by themselves are an industry standard (https://jwt.io/) and there are libraries for creating raw JWT objects, but there is additional complexity in crafting the JWT to the expectations of Vonage. By offering a standalone JWT package per-language, we can offer customers a quick development win by making it easy to generate Vonage-compatible JWTs with our requirements, and continue to allow developers and customers the ability to not have to require a “kitchen sink” SDK for one small operation.
+
+## Proposed Solution
+### Packaging
+Vonage will provide a self contained JWT token generator for each language, that is automatically installed in the SDKs but can also be installed via each language’s package manager. The package will be stored as an open-source project on Github under `vonage/vonage-jwt-[language]` and will be maintained with the same level of maintenance we provide the full server SDKs.
```
Comment (superchilled) [inline rfcs/proposed/RFC-5-server-sdk-jwt-packages.md:32]: This bit we've already done, right? E.g. https://github.com/Vonage/vonage-jwt-ruby

File: rfcs/proposed/RFC-5-server-sdk-jwt-packages.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+* - Machine Names may be converted to camelCase, snake_case, or PascalCase as appropriate for the implementing language.
+
+### Signature Verification
+
+Vonage supports signed callbacks via a supplied JWT on incoming webhook/callback payloads. Users MUST be able to verify these JWTs to be able to correctly validate that an incoming payload has originated from the Vonage API servers.
+
+For any service that supports signed callbacks, the incoming request to the customer application MUST contain an `Authorization` header with a `Bearer` token. The user MUST extract the token from the `Authorization` header themselves, and MUST extract the raw string token from the header so that it does not contain the `Bearer` header.
+
+The SDK implementation MUST include a new method to validation the signature string, and this method SHOULD be named `verifySignature`. It MUST take two parameters, the first being a string representation of the token in base64 format, and the second parameter being the customer's string Secret Key. The method MUST return a boolean value.
```
Comment (superchilled) [inline rfcs/proposed/RFC-5-server-sdk-jwt-packages.md]:
```suggestion
The SDK implementation MUST include a new method to validate the signature string, and this method SHOULD be named `verifySignature`. It MUST take two parameters, the first being a string representation of the token in base64 format, and the second parameter being the customer's string Secret Key. The method MUST return a boolean value.
```

File: rfcs/proposed/RFC-5-server-sdk-jwt-packages.md (hunk 1 above)
Comment (dragonmantank) [reply rfcs/proposed/RFC-5-server-sdk-jwt-packages.md:32]: Correct. Everything outside of the new "Signature Verification" is staying the same

File: rfcs/proposed/RFC-5-server-sdk-jwt-packages.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+The method MUST default to Sha256 as the signature method, as defined by the Vonage Secure Callbacks implementation.
```
Comment (superchilled) [inline rfcs/proposed/RFC-5-server-sdk-jwt-packages.md]: More specifically, isn't it SHA-256 HMAC?

Comment (dragonmantank) [reply rfcs/proposed/RFC-5-server-sdk-jwt-packages.md]: Yup, updated

File: rfcs/proposed/RFC-5-server-sdk-jwt-packages.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+
+## Proposers
+Chris Tankersley
+
+## Detail
+JWT authentication is being used on newer APIs being deployed by Vonage as it provides a finer grain of control over authorization between clients. New applications being developed using APIs such as Messages or Conversations are increasingly being developed by outside teams such as Pre-Sales, and there is a large need for generating JWTs easily. Vonage currently offeres two solutions:
+
+* `vonage/cli`, which can generate JWTs
+* `vonage-java-jwt`, which can generate JWTs in Java
```
Comment (SMadani) [inline rfcs/proposed/RFC-5-server-sdk-jwt-packages.md]:
```suggestion
* `vonage-jwt-jdk`, which can generate JWTs in JVM-based languages ([link to repo](https://github.com/Vonage/vonage-jwt-jdk))
```

File: rfcs/proposed/RFC-5-server-sdk-jwt-packages.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+## Current State
+Proposal
+
+
+## Proposers
+Chris Tankersley
+
+## Detail
+JWT authentication is being used on newer APIs being deployed by Vonage as it provides a finer grain of control over authorization between clients. New applications being developed using APIs such as Messages or Conversations are increasingly being developed by outside teams such as Pre-Sales, and there is a large need for generating JWTs easily. Vonage currently offeres two solutions:
```
Comment (SMadani) [inline rfcs/proposed/RFC-5-server-sdk-jwt-packages.md]:
```suggestion
JWT authentication is being used on newer APIs being deployed by Vonage as it provides a finer grain of control over authorization between clients. New applications being developed using APIs such as Messages or Conversations are increasingly being developed by outside teams such as Pre-Sales, and there is a large need for generating JWTs easily. Vonage currently offers two solutions:
```

**Pull request merged**